reg_tests/CTestList.cmake

```diff
@@ -238,6 +238,7 @@ else(NOT ENABLE_CUDA)
   add_test_r(ablNeutralNGPTrilinos 2)
   add_test_r(conduction_p4 2)
   add_test_r(airfoilRANSEdgeNGPTrilinos.rst 1)
+  add_test_r(ActLineSimpleNGP 4)
```

Comment:
Please use only 2 GPUs, and convert the CPU test to be also 2 MPI ranks. We are not currently setup to assign multiple MPI ranks per GPU with our test setup.

Reply:
Done